Framework/Algorithms/src/NormaliseByCurrent.cpp

```diff
@@ -42,15 +44,14 @@ double NormaliseByCurrent::extractCharge(
   int nPeriods = 0;
   try {
     Property *nPeriodsProperty = run.getLogData("nperiods");
-    Kernel::toValue<int>(nPeriodsProperty->value(), nPeriods);
+    nPeriods = boost::lexical_cast<int>(nPeriodsProperty->value());
```

Comment:
with c++11 you can also now do this with [tostring](http://en.cppreference.com/w/cpp/string/basic_string/to_string) but I don'r recommend changing anything.